packages/blocks/src/__internal__/rich-text/rich-text-operations.ts

```diff
@@ -159,11 +159,14 @@ export function handleLineStartBackspace(store: Store, model: ExtendedModel) {
       if (!parent || parent?.flavour === 'group') {
         const container = getContainerByModel(model);
         const previousSibling = getPreviousBlock(container, model.id);
-        if (previousSibling) {
+        if (previousSibling && previousSibling.flavour === 'paragraph') {
           store.captureSync();
+          let preTextLength = model.text?.length || 0;
           previousSibling.text?.join(model.text as Text);
           store.deleteBlock(model);
           asyncFocusRichText(store, previousSibling.id);
```

Comment:
Can we switch current `asyncFocusRichText` implementation to your `quill.setSelection` approach, or just remove this line? We shouldn't set cursor twice with 2 different APIs.

Reply:
See comment in #162